docs/package_index_json-specification.md

```diff
@@ -247,13 +247,15 @@ Each PLATFORM describes a core for a specific architecture. The fields needed ar
   platform. Each tool is referenced by the triple (`packager`, `name`, `version`) as previously said. Note that you can
   reference tools available in other packages as well, even if no platform of that package is installed.
 - `discoveryDependencies`: the Pluggable Discoveries needed by this platform. Each discovery is referenced by the pair
-  (`packager`, `name`), the `version` is not specified because the latest installed discovery tool will always be used.
-  Like `toolsDependencies` they will be installed by Boards Manager along with the platform and can reference tools
-  available in other packages as well, even if no platform of that package is installed.
+  (`packager`, `name`). The Pluggable Discoveries must be part of `tools` section of the platform. The `version` is not
```

Comment:
I'm trying to explain `discoveryDependencies`, `monitorDependencies` and `toolDependencies` all refer to the same set of tools defined in the `tools` section of the package_index.json.

It's intuitive that the `toolsDependencies` field is a reference to the tools defined in the `tools` section, but someone that reads the specification for the first time may think that the `discoveryDependencies` field may reference a hypothetical `disocveries` section.

Reply:
>tools defined in the `tools` section of the package_index.json

That is not in the platform (i.e., there is no such thing as `packages[].platforms[].tools[]`). It is in the package (i.e., `packages[].tools[]`). Saying `tools` is a section of the platform was the main source of my confusion.

The other thing confusing to me is "the platform", which to me means the same platform as this `discoveryDependencies` definition. That contradicts the later statement:

> can reference tools available in other packages as well

---

How about this?:

```markdown
- `discoveryDependencies`: the Pluggable Discoveries needed by this platform. These are [tools](#tools-definitions),
  defined exactly like the ones referenced in `toolsDependencies`. Unlike `toolsDependencies`, discoveries are
  referenced by the pair (`packager`, `name`). The `version` is not specified because the latest installed discovery
  tool will always be used. Like `toolsDependencies` they will be installed by Boards Manager along with the platform
  and can reference tools available in other packages as well, even if no platform of that package is installed.
```